ch_pipeline/core/io.py

```diff
@@ -473,3 +472,4 @@ def next(self, files):
         )
 
         return sorted(list(new_files))
+
```

Comment:
I think these new lines can be be removed so that this PR does not modify the `io` module.